build.sbt

```diff
@@ -535,7 +535,7 @@ lazy val testSettings: Seq[Def.Setting[_]] = List(
       // Enable verbose logging using sbt loggers in CI.
       List(Tests.Argument(TestFrameworks.MUnit, "+l", "--verbose", "-F"))
     } else {
-      Nil
+      List(Tests.Argument(TestFrameworks.MUnit, "-F"))
```

Comment:
I would advise against this. Full stack traces are HUGE most of the time making it much harder to make sense of normal errors. The only time I have needed a full stack trace was to fix a bug in the scalacheck-munit integration. If there's a bug in how traces are trimmed then it would be best to fix them upstream https://github.com/scalameta/munit/blob/35d868403aca0e71b31b8a794472ddcbee370334/munit/shared/src/main/scala/munit/internal/console/StackTraces.scala

Alternatively, it could be that the exception is happening inside a custom test/value transform 🤔